package.json

```diff
@@ -34,5 +34,7 @@
     "lodash": "^4.17.11",
     "minimatch": "^3.0.4"
   },
-  "dependencies": {}
+  "dependencies": {
+    "bootstrap": "^4.1.3"
```

Comment:
You don't need install Boostrap as dependency if you don't import it through node_modules, so remove it please :)

Reply:
Sure. Thanks for the review. I will get it done quick